.clang-format

```diff
@@ -20,7 +20,6 @@ BinPackArguments: true
 BinPackParameters: true
 BraceWrapping:
   AfterClass: true
-  AfterCaseLabel: true
```

Comment:
Why change this?

Reply:
Forgot to remove it from the commit, I need that to get clang-format to work.